src/containers/Auth/Login.jsx

```diff
@@ -0,0 +1,80 @@
+import React, { useContext, useEffect } from "react";
+import * as Yup from "yup";
+import AppFormField from "../../components/Form/AppFormField";
+import SubmitButton from "../../components/Form/SubmitButton";
+import AppForm from "../../components/Form/AppForm";
+import Grid from "@material-ui/core/Grid";
+import Block from "../../components/Block/Block";
+import { useHistory } from "react-router-dom";
+import Button from "@material-ui/core/Button";
+import { makeStyles } from "@material-ui/core/styles";
+import { AuthContext } from "../../context/AuthContext";
+
+const validationSchema = Yup.object().shape({
+  email: Yup.string().required().email().label("Email"),
+  password: Yup.string().required().label("Password"),
+});
+
+const useStyles = makeStyles((theme) => ({
+  button: {
+    padding: theme.button.padding,
+  },
+}));
+
+const Login = () => {
+  const history = useHistory();
+  const classes = useStyles();
+  const { state: authState, signIn } = useContext(AuthContext);
+
+  const handleSubmit = (value) => {
+    signIn(value);
+  };
+
+  return (
+    <Block lg={4} md={6} sm={8} xs={11}>
+      <AppForm
+        initialValues={{ email: "test@gmail.com", password: "123" }}
+        onSubmit={handleSubmit}
+        validationSchema={validationSchema}
+      >
+        <Grid container spacing={2}>
+          <Grid item md={12} sm={12} xs={12}>
```

Comment:
when using same spaces let's keep them in the theme file as constants since we are using them all around this file